src/modules/logger/logger.cpp

```diff
@@ -536,57 +536,61 @@ void Logger::add_common_topics()
 	add_topic("actuator_controls_0", 100);
 	add_topic("actuator_controls_1", 100);
 	add_topic("actuator_outputs", 100);
-	add_topic("airspeed");
+	add_topic("airspeed", 200);
+	add_topic("arm_auth_ack");
+	add_topic("arm_auth_request");
```

Comment:
I'm wondering if we really need the `arm_auth_*` topics, since these are pre-arm, and logging is only started upon arming in general.

Reply:
Hmm, I don't even see these as uORB messages. These must be vehicle commands?

Reply:
Indeed they don't exist - not even as multi-topics. I removed them in https://github.com/PX4/Firmware/commit/201b2bd75eba41a1daa78f7f950aea6ce6e20acf